easybuild/easyconfigs/g/GLM/GLM-0.9.9.8.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'GLM'
+version = '0.9.9.8'
+
+homepage = 'https://github.com/g-truc/glm'
+description = """
+OpenGL Mathematics (GLM) is a header only C++ mathematics library for graphics  
+software based on  the OpenGL Shading Language (GLSL) specifications."""
+
+toolchain = SYSTEM  # header only library
+
+source_urls = ['https://github.com/g-truc/glm/archive/']
+sources = ['%(version)s.tar.gz']
+patches = [
+    'GLM-0.9.9.8_fix_missing_install.patch',
+]
+checksums = [
+    '7d508ab72cb5d43227a3711420f06ff99b0a0cb63ee2f93631b162bfe1fe9592',  # 0.9.9.8.tar.gz
+    '1cc199f9d66679b0b5e9a9fbe20bca0d9b15760fa172ca8759dd15bab35802ca',  # GLM-0.9.9.8_fix_missing_install.patch
+]
+
+builddependencies = [
+    ('GCCcore', '10.2.0'),
+    ('binutils', '2.35'),
+    ('CMake', '3.18.4', '-GCCcore-10.2.0'),
+]
```

Comment:
I understand this is header only, but we would like to avoid this as it can cause some issues with HMNS and such. Placing it under GCCcore level instead, like we have done for Eigen.

This isn't so bad though; the GCCcore/toochain bumps offers a regular opportunity to update versions of all libraries, with a better checking what (mostly single) version if used. 

Long story short: please use GCCcore toolchain instead